filebeat/generator/fields/fields.go

```diff
@@ -179,15 +179,12 @@ func addNewField(fs []field, f field) []field {
 	return append(fs, f)
 }
 
-func getSemanticElementsFromPatterns(patterns []string) ([]field, error) {
-	r, err := regexp.Compile("{[\\.\\w\\:]*}")
-	if err != nil {
-		return nil, err
-	}
+var semanticElementsRegex = regexp.MustCompile(`{[\.\w\:]*}`)
```

Comment:
I assume this is the mentioned performance improvement. While I agree that this might improve performance, without benchmarks and numbers we cannot claim anything.

I propose we either: 

1. change the commit message / PR description and keep just the clean-up part
2. or we add a benchmark for `getSemanticElementsFromPatterns` and add results before and after this change to the PR description.

Reply:
@rdner I have update the commit message and removed the mention of performance improvements.